editors/vscode/rolldown.config.test.ts

```diff
@@ -0,0 +1,20 @@
+import { glob } from 'glob';
+import type { RolldownOptions } from 'rolldown';
+
+export default async (): Promise<RolldownOptions> => {
```

Comment:
please use the `defineConfig` syntax.
`import { defineConfig } from 'rolldown';`

use `globSync` from `tinyglobby` and ignore the bot :)